source/voice_control/s3-box-customize.markdown

```diff
@@ -0,0 +1,189 @@
+---
+
+This tutorial will show you how to replace the Home Assistant status images on the ESP32-S3-BOX by custom images. 
+
+You can either prepare your own images or use images from a community repository.
+It applies to the following three ESP32 devices by Espressif:
+
+- [ESP32-S3-BOX](https://www.espressif.com/en/news/ESP32-S3-BOX_video)
+- [ESP32-S3-BOX-3](https://www.espressif.com/en/news/ESP32-S3-BOX-3)
+- [ESP32-S3-BOX-Lite](https://www.espressif.com/en/news/ESP32-S3-BOX_video)
+
+The steps are the same for all three product variants. The term *ESP32-S3-BOX* is used to refer to all three variants.
+
+## Prerequisites
+
+- Latest version of Home Assistant, installed with the Home Assistant Operating System
+- [Home Assistant Cloud](https://www.nabucasa.com) or a manually configured [Assist Pipeline](/voice_control/voice_remote_local_assistant)
+- Successfully completed the [ESP32-S3-BOX voice assistant](/voice_control/s3_box_voice_assistant/) tutorial
+
+## Adopting the device in the ESPHome add-on
+
+Before you can import new images, you need to install the ESPHome add-on and adopt the device in the add-on.
+
+1. Make sure the ESP32-S3-BOX is up and running and connected to your Wi-Fi.
+2. Go to **Settings** > **Add-ons** and check if you have the **ESPHome** add-on installed.
+   - If you haven't done so, install the add-on.
+3. Start the add-on.
+4. Once the add-on is running, select **Open web UI**.
+5. In the ESPHome add-on dashboard, on the **ESP32-S3-BOX** card, select **Adopt**.
+   ![Adopt the ESP32-S3-BOX in the ESPHome add-on](/images/assist/esp32-adopt-s3.png)
+6. If you like, give it a new name. Then, select **Adopt**.
+7. Copy the encryption key and store it in a safe place.
+   - It will also be available within the UI, but it is always good to keep a copy.
+   - **Result**: Once you copied the key, the status will change to **Online**.
+8. Now that you have set up the ESPHome add-on, continue with one of the 2 different methods to add custom images:
+   - [Using images from a community repository](#using-images-from-a-community-repository)
+   - [Using your own images](#using-your-own-images)
+
+## Using images from a community repository
+
+- Perform this procedure, if you want to use ready-made images by the community.
+- If you want to use your own images, skip this procedure and go to [Using your own images](#using-your-own-images) instead.
+
+**Duration**: about 30 minutes
+
+### To use images from the community
+
+1. On the **ESP32-S3-BOX** add-on, select edit.
+   - **Result**: An editor opens, showing the configuration file.
+   ![ESP32-S3-BOX config file](/images/assist/esp32-adopt-s3-01.png)
+2. For inspiration, we have prepared some images for you.
+   - Check them out in this [repository](https://github.com/jlpouffier/home-assistant-s3-box-community-illustrations/tree/main).
+3. For this tutorial, we will use some images of Frenck.
```

Comment:
should we link to the Frenck folder so user can see from the readme where the below values comes from ?